product_supplierinfo_archive/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html)
+
+{
+    "name": "Product Supplier Info Archive",
```

Comment:
```suggestion
    "name": "Product Supplier Info Active",
```
and module would be product_supplierinfo_active.

Shouldn't this be a better name? I mean, the field added is 'active'

Reply:
Yes, maybe it would be a better name. But in the beginning i based the name of the module on others like this one: https://github.com/OCA/product-attribute/tree/14.0/product_attribute_archive

Reply:
Oh, then good :+1: